File: Python/Arrays/05.ReverseArray.py
def revArray(array):
    start = 0
    end = len(array) -1

    while start<=end:
        a = array[start]
        array[start] = array[end]
        array[end] = a

        start += 1
        end -= 1

    for i in range (len(array)):
        print(array[i], end=' ')



arr = [1, 2, 5, 6, 9, 11, 23, 78, 32, 12]

revArray(arr)

File: CPP/Arrays/05.ReverseArray.cpp
#include <bits/stdc++.h>
using namespace std;

void revArray(int arr[], int size){

    int start = 0;
    int end = size-1;

    while (start<=end)
    {
        swap(arr[start], arr[end]);
        
        start++;
        end--;
    }
    
    for (int i = 0; i < size; i++)
    {
        cout<<arr[i]<<" ";
    }
    
}

int main(){

    int anArray[10] = {1, 2, 5, 6, 9, 11, 23, 78, 32, 12};

    int size = 10;

    revArray(anArray, size);

    return 0;
}